cgmanifests/submodules/cgmanifest.json

```diff
@@ -335,52 +335,12 @@
       "component": {
         "type": "git",
         "git": {
-          "commitHash": "99296a4462ffc0a8a4a9635e51382277bec68979",
+          "commitHash": "53ed9b36050bc48e4e6fcc1996fb51c408c066b5",
           "repositoryUrl": "https://github.com/onnx/onnx-tensorrt.git"
         },
         "comments": "git submodule at cmake/external/onnx-tensorrt"
       }
     },
-    {
```

Comment:
are those supposed to be removed?
seems like it no longer recurses into submodules? just wanted to check if this is expected. @snnn

Reply:
No. I guess Steven  didn't run 
```
git submodule update --init --recursive
```

Reply:
I do run the git update

Reply:
It turns out I need to run the script from root dir. Has rerun it and fixed the issue